Thing.h

```diff
@@ -695,7 +695,7 @@ class ThingDevice {
 #endif
   }
 
-  void serialize(JbsonObject descr, String ip, uint16_t port) {
+  void serialize(JsonObject descr, String ip, uint16_t port) {
```

Comment:
ok please squash commit in previous one there is no value in keeping the mistake

if more changes can be squashed please do